scripts/passiv/passiv_data.py

```diff
@@ -0,0 +1,77 @@
+""" Script to merge Passiv data, ready for nowcasting"""
+import time
+
+import numpy as np
+import pandas as pd
+
+dir = "gs://solar-pv-nowcasting-data/PV/Passive/20211027_Passiv_PV_Data/5min"
+file_output = "../passive.netcdf"
+
+t = time.time()
+
+months = pd.date_range("2021-01-01", "2021-12-01", freq="MS").strftime("%b").tolist()
+years = ["2020", "2021"]
+
+
+def load_file():
+    """
+    Load all passiv data
+
+    Join years and months together
+    """
+
+    data_df = []
+    for year in years:
+        for month in months:
+            file = f"{dir}/{year}/{month}.csv.gz"
+            print(file)
+            try:
+                df = pd.read_csv(file)
+                data_df.append(df)
+            except Exception as e:  # noqa F481
+                print(f"Could not load {file}")
```

Comment:
Do we also want to re-raise this exception, so the script crashes if we can't load this file?

If not, please add a comment to explain why we don't want to crash.

Reply:
ah yea Ill add a comment. Its bascially becasue 2021-Nov and 2021-Dec isnt there, but I didnt want it to break becasue of that